app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -1,25 +1,19 @@
 # rubocop:disable Naming/AccessorMethodName
 module ManageIQ::Providers::Lenovo
   class PhysicalInfraManager::RefreshParser < EmsRefresh::Parsers::Infra
-    include ManageIQ::Providers::Lenovo::RefreshHelperMethods
-
     def self.miq_template_type
       parent::Parser::ParserDictionaryConstants::MIQ_TYPES["template"]
     end
 
-    def initialize(ems, options = nil)
```

Comment:
Why did you get rid of options?  Even if it isn't used today it might be helpful in the future.

Reply:
I just removed because It is unused. I will set the variable as `_options` then.